commit dbf64e83af8d4b5a55fed546e0c615cd51141bd7
Author: Lawrence Velázquez <larryv@macports.org>
Date:   Tue Nov 25 01:50:58 2014 -0500

    Include space.h in deprecated headers using isl_dim_type
    
    These headers should not assume that space.h will be included
    transitively through another header or by an including source file, as
    compilation can fail if those assumptions don't hold.
    
    Signed-off-by: Lawrence Velázquez <larryv@macports.org>

Index: include/isl/deprecated/aff_int.h
===================================================================
--- include/isl/deprecated/aff_int.h.orig
+++ include/isl/deprecated/aff_int.h
@@ -3,6 +3,7 @@
 
 #include <isl/deprecated/int.h>
 #include <isl/aff_type.h>
+#include <isl/space.h>
 
 #if defined(__cplusplus)
 extern "C" {
Index: include/isl/deprecated/constraint_int.h
===================================================================
--- include/isl/deprecated/constraint_int.h.orig
+++ include/isl/deprecated/constraint_int.h
@@ -3,6 +3,7 @@
 
 #include <isl/deprecated/int.h>
 #include <isl/constraint.h>
+#include <isl/space.h>
 
 #if defined(__cplusplus)
 extern "C" {
Index: include/isl/deprecated/map_int.h
===================================================================
--- include/isl/deprecated/map_int.h.orig
+++ include/isl/deprecated/map_int.h
@@ -3,6 +3,7 @@
 
 #include <isl/deprecated/int.h>
 #include <isl/map_type.h>
+#include <isl/space.h>
 
 #if defined(__cplusplus)
 extern "C" {
Index: include/isl/deprecated/point_int.h
===================================================================
--- include/isl/deprecated/point_int.h.orig
+++ include/isl/deprecated/point_int.h
@@ -3,6 +3,7 @@
 
 #include <isl/deprecated/int.h>
 #include <isl/point.h>
+#include <isl/space.h>
 
 #if defined(__cplusplus)
 extern "C" {
Index: include/isl/deprecated/polynomial_int.h
===================================================================
--- include/isl/deprecated/polynomial_int.h.orig
+++ include/isl/deprecated/polynomial_int.h
@@ -3,6 +3,7 @@
 
 #include <isl/deprecated/int.h>
 #include <isl/polynomial.h>
+#include <isl/space.h>
 
 #if defined(__cplusplus)
 extern "C" {
Index: include/isl/deprecated/set_int.h
===================================================================
--- include/isl/deprecated/set_int.h.orig
+++ include/isl/deprecated/set_int.h
@@ -3,6 +3,7 @@
 
 #include <isl/deprecated/int.h>
 #include <isl/set_type.h>
+#include <isl/space.h>
 
 #if defined(__cplusplus)
 extern "C" {
